modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/test/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContextTest.java

```diff
@@ -129,6 +130,40 @@ public void testGetFormURLForSharedFormInstance() throws Exception {
 		Assert.assertEquals(getSharedFormURL(), formURL);
 	}
 
+	@Test
+	public void testGetPublishedURLForRestrictedFormInstance()
+		throws Exception {
+
+		DDMFormInstance ddmFormInstance = mockDDMFormInstance(
+			_SHARED_FORM_INSTANCE_ID, false);
+
+		String publishedFormURL =
+			_ddmFormAdminDisplayContext.getPublishedFormURL(ddmFormInstance);
+
+		Assert.assertEquals(StringPool.BLANK, publishedFormURL);
+	}
```

Comment:
Hey @marcosapmf, this test intends to check the scenario in which the form isn't published and hence there is no published URL, right? If so, can you please change the test name to make it more explicit? For example, `testGetPublishedURLForUnpublishedForm`.

In addition, just reading this test isn't clear that, by default, the form isn't published, therefore would be nice you create a method to mock this setting and re-use it on the two tests created by you on this class, can you please do this?

Thanks.

Reply:
Sure, @natocesarrego!

Thank you.